src/components/ChatBotInput/ChatBotInput.tsx

```diff
@@ -187,13 +188,19 @@ const ChatBotInput = ({
 				onBlur={handleBlur}
 			/>
 			<div className="rcb-chat-input-button-container">
-				{!botOptions.voice?.disabled && isDesktop &&
-					<VoiceButton inputRef={inputRef} textAreaDisabled={textAreaDisabled}
-						voiceToggledOn={voiceToggledOn} handleToggleVoice={handleToggleVoice}
-						triggerSendVoiceInput={triggerSendVoiceInput} setInputLength={setInputLength}
-					/>
-				}
-				<SendButton handleSubmit={handleSubmit}/>
+				{botOptions.chatInput?.buttons?.map((button, index) => {
+					switch(button){
+					case "send":
+						return <SendButton key={index} handleSubmit={handleSubmit}/>
+					case "voice":
+						return !botOptions.voice?.disabled && isDesktop &&
+							<VoiceButton key={index} inputRef={inputRef} textAreaDisabled={textAreaDisabled}
+								voiceToggledOn={voiceToggledOn} handleToggleVoice={handleToggleVoice}
+								triggerSendVoiceInput={triggerSendVoiceInput} setInputLength={setInputLength}
+							/>
+					}
+					return <React.Fragment key={index}>{button}</React.Fragment>
+				})}
```

Comment:
Logic is sound, but just a small nit - prefer to have consistent styling across the codebase. The [header](https://github.com/tjtanjin/react-chatbotify/pull/38)/[footer](https://github.com/tjtanjin/react-chatbotify/pull/33/files) implementations renders the individual buttons separately and uses `if-else` instead of `switch-case`. Since what is checked in each block involves more than just a simple variable change, `if-else` is preferred.

Reply:
fixed

Reply:
> fixed

I saw you're using `if-else` now, but the rendering of the buttons should also be handled separately. Similar to `renderNotificationButton` that you can find here: https://github.com/tjtanjin/react-chatbotify/pull/38/files. Lastly, there are missing whitespaces in your `if-else` statements. For example:
- `if(button === "send"){` (current)
- `if (button === "send") {` (preferred)

The same applies to the rest of the conditional statements. Trouble you to help make the changes, thanks!

Reply:
Fix. Let me know if there's anything else.